cartographer_ros/cartographer_ros/submap.cc

```diff
@@ -30,29 +30,28 @@ std::unique_ptr<SubmapTexture> FetchSubmapTexture(
   ::cartographer_ros_msgs::SubmapQuery srv;
   srv.request.trajectory_id = submap_id.trajectory_id;
   srv.request.submap_index = submap_id.submap_index;
-  if (!client->call(srv)) {
-    return nullptr;
-  }
-  std::string compressed_cells(srv.response.cells.begin(),
-                               srv.response.cells.end());
+  CHECK(client->call(srv)) << "SubmapQuery call failed.";
```

Comment:
This is out of the control of the caller, i.e. an expected failure. Do not CHECK here.